Order id is 210204000132622 and order Variety is co
Order id is 210204000132624 and order Variety is co
unable to delete order id :  210204000132624
Order id is 210204000132625 and order Variety is co
Order id is 210204000132635 and order Variety is co
Order id is 210204000132637 and order Variety is co
unable to delete order id :  210204000132637
Order id is 210204000132639 and order Variety is co
Order id is 210204000132640 and order Variety is co
unable to delete order id :  210204000132640
Order id is 210204000132761 and order Variety is co
Order id is 210204000132857 and order Variety is co
Order id is 210204000132999 and order Variety is co
Order id is 210204000132859 and order Variety is co
Order id is 210204000727384 and order Variety is regular
Order id is 210204000732554 and order Variety is regular
Order id is 210204002478030 and order Variety is regular
Order id is 210204002478043 and order Variety is regular
Order id is 210204002478050 and order Variety is regular
Order id is 210204002478052 and order Variety is regular
Order id is 210204002478054 and order Variety is regular
Order id is 210204002478055 and order Variety is regular
Order id is 210204002478098 and order Variety is regular
Order id is 210204002478103 and order Variety is regular
Order id is 210204002478104 and order Variety is regular
Order id is 210204002478106 and order Variety is regular
Order id is 210204002478248 and order Variety is regular
Order id is 210204002478258 and order Variety is regular
Order id is 210204002478262 and order Variety is regular
Order id is 210204000132624 and order Variety is co
unable to delete order id :  210204000132624
Order id is 210204000132637 and order Variety is co
unable to delete order id :  210204000132637
Order id is 210204000132640 and order Variety is co
unable to delete order id :  210204000132640
Quantities for buy to be placed 150
Traceback (most recent call last):
  File "square_off.py", line 89, in <module>
  File "square_off.py", line 32, in placeMarketOrder
    transaction_type=t_type,
  File "/Users/saravana.kumar/opt/anaconda3/envs/algo/lib/python3.8/site-packages/kiteconnect/connect.py", line 346, in place_order
    return self._post("order.place", params)["order_id"]
  File "/Users/saravana.kumar/opt/anaconda3/envs/algo/lib/python3.8/site-packages/kiteconnect/connect.py", line 821, in _post
    return self._request(route, "POST", params)
  File "/Users/saravana.kumar/opt/anaconda3/envs/algo/lib/python3.8/site-packages/kiteconnect/connect.py", line 886, in _request
    raise exp(data["message"], code=r.status_code)
kiteconnect.exceptions.InputException: Unknown `tradingsymbol`.
